### DISCUSSION TITLE
How do you install it?

### DISCUSSION THREAD

**Discussion by SindarinLord** [Tyrrrz/DiscordChatExporter#574] [Q&A]
I am new to GitHub and really want this so how do I install it

**Comment by Tyrrrz** [comment]
Go to the [download](https://github.com/Tyrrrz/DiscordChatExporter#download) section of the readme, find the appropriate flavor (if you're on Windows then GUI is recommended, otherwise use CLI), then go to "stable release" an download it. Remember to also install .NET runtime if you haven't yet (the link is in the table).
